test/ChildMapping-test.tsx

```diff
@@ -32,102 +27,102 @@ describe('ChildMapping', () => {
 
   it('should support mergeChildMappings for adding keys', () => {
     let prev = {
-      one: true,
-      two: true,
+      one: 'one',
```

Comment:
This looks like runtime behavior changed. But the PR title says it's only about test. Could you clarify why this change here is necessary?

Reply:
The type is `ReactChild`, which doesn't accept `boolean`, so I've changed the test data to String.

Reply:
Sure but that's not a reason to change runtime behavior, no? So far this looks like a breaking change just so that our tests are type-checked?

Reply:
What does "runtime behavior" mean? This doesn't change any runtime behavior. The TypeScript migration causes this change. Do you mean that we shouldn't do any incompatible changes with DefinitelyTyped?